DynamicTablesPkg/Library/Common/SsdtPcieSupportLib/SsdtPcieOscTemplate.asl

```diff
@@ -46,21 +46,16 @@ DefinitionBlock ("SsdtPciOsc.aml", "SSDT", 2, "ARMLTD", "PCI-OSC", 1) {
 
       // Save Capabilities DWord2 & 3
       Store (CDW2, SUPP)
-      Store (CDW3, CTRL)
+      And (CTRL, CDW3, CTRL)
 
       // Only allow native hot plug control if OS supports:
       // * ASPM
       // * Clock PM
       // * MSI/MSI-X
       If (LNotEqual (And (SUPP, 0x16), 0x16)) {
-        And (CTRL, 0x1E, CTRL) // Mask bit 0 (and undefined bits)
+        And (CTRL, 0x9E, CTRL) // Mask bit 0 (and undefined bits)
       }
 
-      // Always allow native PME, AER (no dependencies)
-
-      // Never allow SHPC (no SHPC controller in this system)

```

Comment:
Same here, can we please add a `MU_CHANGE` tag and just comment out this section?

Reply:
Added MU_CHANGE tags to all added code, thanks Kun